src/api/middlewares/errorHandler.js

```
@@ -4,7 +4,7 @@ const notFound = (req, res, next) => {
     next(error)
 }
 
-const errorHandler = (err, req, res, next) => {
+const errorHandler = (err, req, res,) => {
     const statusCode = res.statusCode === 200 ? 500 : res.statusCode
```

What is going on here?
Why are you substituting 200 for 500?

---

The logic behind is that when an error gets to this handler without a statuscode it will default to a 200 which is misleading. So instead default to 500 if no status code was set before this error handler. so this ensures we dont send a 200 for an error.

---

the question is how does status code become 200 in the first place, and why is 200 entering the errorhandler function?

---

resolved